eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2.java

```diff
@@ -75,23 +76,48 @@ public RegionWSServerV2() {
-                                              @RequestParam(name = "studies", required = false) List<String> studies,
-                                              @RequestParam(name = "annot-ct", required = false) List<String>
-                                                      consequenceType,
-                                              @RequestParam(name = "maf", required = false) String maf,
-                                              @RequestParam(name = "polyphen", required = false) String polyphenScore,
-                                              @RequestParam(name = "sift", required = false) String siftScore,
-                                              @RequestParam(name = "annot-vep-version", required = false) String
-                                                      annotationVepVersion,
-                                              @RequestParam(name = "annot-vep-cache-version", required = false) String
-                                                      annotationVepCacheVersion,
-                                              @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
-                                              @RequestParam(required = false, defaultValue = "20") Integer pageSize,
-                                              HttpServletResponse response,
-                                              @ApiIgnore HttpServletRequest request)
+    public ResponseEntity getVariantsByRegion(
+            @ApiParam(value = "Comma separated genomic regions in the format chr:start-end.")
+            @PathVariable("regionId") String regionId,
+            @ApiParam(value = "First letter of the genus, followed by the full species name, e.g. hsapiens. " +
+                    "Allowed values can be looked up in /v1/meta/species/list/ in the field named 'taxonomyCode'.",
+                    required = true)
+            @RequestParam(name = "species") String species,
+            @ApiParam(value = "Encoded assembly name, e.g. grch37. Allowed values can be looked up in " +
+                    "/v1/meta/species/list/ in the field named 'assemblyCode'.", required = true)
+            @RequestParam(name = "assembly") String assembly,
+            @ApiParam(value = "Identifiers of studies. If this field is null/not specified, all studies should" +
+                    " be queried. Each individual identifier of studies can be looked up in" +
+                    " /v2/studies in the field named `studyId`. e.g. PRJEB6930,PRJEB27824")
+            @RequestParam(name = "studies", required = false) List<String> studies,
+            @ApiParam(value = "Retrieve only variants with exactly this consequence type (as stated by Ensembl VEP)")
+            @RequestParam(name = "annot-ct", required = false) List<String>
+                    consequenceType,
+            @ApiParam(value = "Filter variants using Minor Allele Frequency. If only the value is given, it compares" +
```

Comment:
To follow the style of the previous ones, where the meaning is introduced without any preamble.
```suggestion
            @ApiParam(value = "Minor Allele Frequency. If only the value is given, it compares" +
```